x-pack/plugins/cases/server/client/configure/client.ts

```diff
@@ -325,7 +319,7 @@ async function update(
       configurationId: configuration.id,
       updatedAttributes: {
         ...queryWithoutVersionAndConnector,
-        ...(connector != null ? { connector: transformCaseConnectorToEsConnector(connector) } : {}),
+        ...(connector && { connector }),
```

Comment:
Interesting 🙂 . This is a bit different from the `connector != null ? { connector } : {}` as it will check for all falsy values like `"", 0 false, ...` and not only `null` or `undefined`. Though TS will guard for the other falsy values.

Reply:
good point I'll add the `!= null`